### PR TITLE
Kcptun (new formula)

### PR DESCRIPTION
**Pull request by huijiewei**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by ilovezfs** [comment]
You'll need to peg each resource to a specific git commit `:revision =>` 

You can use `gdm brew`: https://github.com/sparrc/gdm

File: Formula/kcptun.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    "dscp": 46,
+    "nocomp": false,
+    "acknodelay": false,
+    "nodelay": 0,
+    "interval": 40,
+    "resend": 0,
+    "nc": 0,
+    "sockbuf": 4194304,
+    "keepalive": 10
+}
```
Comment (bfontaine) [inline Formula/kcptun.rb]: Please indent this; `.undent` above will take care of removing the indentation.

File: Formula/kcptun.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        <string>#{var}/log/kcptun.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/kcptun.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_match "kcptun version", shell_output("#{bin}/kcptun_client -v")
```
Comment (bfontaine) [inline Formula/kcptun.rb]: Can’t we write a test that does something more substantial than printing the version?

**Comment by MikeMcQuaid** [comment]
@huijiewei Any news on this?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Pull request closed without merging**